integration/crds/lightweight/DanmEp.yaml

```diff
@@ -1,15 +1,104 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  name: danmeps.danm.k8s.io
+  name: danmeps.danm.io
 spec:
-  scope: Namespaced
-  group: danm.k8s.io
-  version: v1
+  group: danm.io
   names:
     kind: DanmEp
+    listKind: DanmEpList
     plural: danmeps
     singular: danmep
     shortNames:
     - de
     - dep
+  scope: Namespaced
+  versions:
+  - name: v1
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              CID:
+                type: string
+              EndpointID:
+                type: string
+              Host:
+                type: string
+              Interface:
+                properties:
+                  Address:
+                    type: string
+                  AddressIPv6:
+                    type: string
+                  DeviceID:
+                    type: string
+                  MacAddress:
+                    type: string
+                  Name:
+                    type: string
+                  proutes:
+                    additionalProperties:
+                      type: string
+                    type: object
+                  proutes6:
+                    additionalProperties:
+                      type: string
+                    type: object
+                required:
```

Comment:
not all interfaces have both V4 and V6 addresses etc
let's not unnecessairly add restrictions to an API which is only wirtten by code anyway, as it could result in failures in some edge cases

Reply:
I will keep only this tow
```yaml
required:
- MacAddress
- Name
```

Reply:
unless you wanna remove `required` section

Reply:
what I want is the API spec to remain semantics the **exact same** during the upgrade. I'm not really sure based on what you are deciding to remove some of the old rules, and add some previously non-existing designations?